parachain/pallets/ethereum-beacon-client/src/lib.rs

```diff
@@ -175,12 +180,6 @@ pub mod pallet {
 	pub(super) type ExecutionHeaders<T: Config> =
 		StorageMap<_, Identity, H256, ExecutionHeader, OptionQuery>;
 
-	/// Current sync committee corresponding to the active header.
-	/// TODO  prune older sync committees than xxx
-	#[pallet::storage]
-	pub(super) type SyncCommittees<T: Config> =
```

Comment:
Its still useful to keep the sync committees aggregate public key, for the optimization described in https://eth2book.info/bellatrix/part2/building_blocks/signatures/#sync-aggregates and which we talked about on Slack.

Reply:
Yes, would prefer to add it back together with the optimization in next PR.